pkg/kubelet/server/server.go

```diff
@@ -101,6 +102,10 @@ const (
 	checkpointPath      = "/checkpoint/"
 	pprofBasePath       = "/debug/pprof/"
 	debugFlagPath       = "/debug/flags/v"
+	podsPath            = "/pods/"
+	runningPodsPath     = "/runningpods/"
+	healthzPath         = "/healthz/"
+	configzPath         = "/configz"
```

Comment:
For handlers registered in other packages, define public constants there that exactly match the strings actually used to register the handler paths, and use the constant in attrs calculation and when registering the handers:

```go
InstallPathHandler(mux, "/healthz", checks...)
```
```go
m.Handle("/configz", http.HandlerFunc(handle))
```

For handlers registered in this package, define constants here that exactly match the strings actually used to register the webservice paths, and use the constant in attrs calculation and when registering the webservice:

```
	ws.
		Path("/pods").
```
```
	ws.
		Path("/runningpods/").
```

Reply:
Done.